ember-flight-icons/tests/dummy/app/helpers/substring.js

```diff
@@ -1,12 +0,0 @@
-import Helper from '@ember/component/helper';
```

Comment:
@MelSumner if we keep this helper around does that obviate the need to manually change the catalog file? We could use the substring-ed version of the icon name to do the searching against then, or am I misunderstanding how this worked?

Reply:
I'll try it and make sure it's not something that I missed 👍